code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -783,3 +783,60 @@
 	var/location = get_turf(holder.my_atom)
 	for(var/i in 1 to created_volume)
 		new /obj/item/stack/sheet/ashresin(location)
+
+/datum/chemical_reaction/sulfur_trioxide
+	name = /datum/reagent/sulfur_trioxide
+	id = /datum/reagent/sulfur_trioxide
+	results = list(/datum/reagent/sulfur_trioxide = 2)
+	required_reagents = list(/datum/reagent/sulfur = 1, /datum/reagent/oxygen = 3)
+	required_catalysts = list(/datum/reagent/hydrogen = 1)
+
+/datum/chemical_reaction/sulfuric_acid
+	name = /datum/reagent/toxin/acid
+	id = /datum/reagent/toxin/acid
+	results = list(/datum/reagent/toxin/acid = 1)
```

Comment:
serious does acid REALLY demand a whole 2 steps with fussy conversion rates what did it do to deserve this

Reply:
It doesn't really but I figured you, Ynot, and Mqiib would have my head for trying to unnerf chemistry 100%